ac_qv_api.py

```diff
@@ -263,7 +263,8 @@ def draw_nets(self):
 				
 
 	def predict_qs(self, voltages):
-		# requires voltage to be vg, vd
+		# requires voltages is an numpy array of size 
+		# (batch size, input_dimension) to be vg, vd
 		# both vg and vd must be numpy arrays

```

Comment:
You need to delete this line as well, also clarify which dimension is vg and vd. I will merge and modify for now. But make sure the comment is correct next time.